metadata-ingestion/docs/sources/snowflake/snowflake_recipe.yml

```diff
@@ -1,11 +1,8 @@
 source:
   type: snowflake
   config:
-    # This option is recommended to be used for the first time to ingest all lineage
+    # This option is recommended to be used to ingest all lineage
```

Comment:
After patch lineage is supported for snowflake, this line can be updated to  original text - "This option is recommended to be used for the first time to ingest all lineage".
Currently, if this is removed, there is a possibility of overriding past lineage for same table.